### ISSUE TITLE
Metadata elements for the terminology list

### ISSUE THREAD

**Issue by mabablue** [i-adopt/core-group#4]
How can we decide on which metadata elements to use from [MOD](https://docs.google.com/spreadsheets/d/1FKAFLEcalqH_Ty3Z5xD39VelsKIeCc9s1FYxAAH1BXk/edit?usp=sharing) needed for the terminology list? should we create a new repository with issues for each metadata element ?

**Comment by mabablue** [comment]
Comment from JG (email): "that work is done and the MOD authors (includes Clement) did it, and should not be re-done. The main task would be to decide which of their attributes should be required, and which are optional. "

**Comment by mabablue** [comment]
I completely agree on this and already suggested it. But we have to decide on which ones to choose and probably also decide if we need additional ones which could be relevant for our Interoperability Framework (like resource type: is this a model or a list of values)

**Comment by gwemon** [comment]
Issue ticket created https://github.com/i-adopt/terminologies/issues/46
